HHS/OPRE-OPS
# 594 carry forward funds update

## What changed

- Added new field to the CANFiscalYearCarryForward model, replacng amount with "received_amount" and "expected_amount".
- Changed "current_funding" to "received_funding" for CANFiscalYear model.
- Summing up the received_amount & expected_amount for carry forwards into a single value, also have the backend send the text for the carry-forward tag for the front-end, with the correct FY put in it.
- Refactored "carry_over" and "CarryOver" to "carry_forward" and "CarryForward" respectively.
- Added new Carry-Forward tag for can cards that appears if that can has any carry-forward amount in it for the current FY.

## Issue

[Carry-forward Funds Update #594](https://app.zenhub.com/workspaces/opre-ops-62d197b4c468970013bcd2ec/issues/gh/hhs/opre-ops/594)

## How to test

- Log into site
- Go to portfolios
- Choose "Healthy Marriage & Responsible Fatherhood"
- See that there is carry-forward in the portfolio summary bar.
- Scroll down through CANs, and 2 of them have carry-forward tags.

## Screenshots

![362FC414-5BE1-4AAB-BDB8-929F9FE652CD](https://user-images.githubusercontent.com/881707/222155757-efe501bc-a2e5-4aac-a774-4b7f32e3062c.jpeg)

![FB33830D-614D-43D5-AF05-2279C813EB61](https://user-images.githubusercontent.com/881707/222155762-eeb3ade7-cd99-444e-96cb-7d425f91a1b6.jpeg)

## Links
